gradle/verification-metadata.xml

```diff
@@ -2430,7 +2458,9 @@
       </component>
       <component group="org.apache.kerby" name="ldap-backend" version="1.1.1">
          <artifact name="ldap-backend-1.1.1.jar">
-            <sha256 value="d3c030fe9eaeeb8a7b0844a1b48cecef20bc0a8deb5718a6c482bfc325069ee0" origin="Generated by Gradle"/>
+            <sha256 value="d3c030fe9eaeeb8a7b0844a1b48cecef20bc0a8deb5718a6c482bfc325069ee0" origin="Generated by Gradle">
+               <also-trust value="b427406dbe76279cf4fd496af17792c4c39fb6f295480290bb8be987d33cbc20"/>
```

Comment:
Why do we need also-trust values?

Reply:
`gradlew updateShas` did this for me, I don't know why. @breskeby Do you know about this?